### DISCUSSION TITLE
Statements after unconditional return

### DISCUSSION THREAD

**Discussion by mark-wiemer** [FuPeiJiang/ahk_explorer#4] [Q&A] [answered]
Hi @FuPeiJiang, as you know I'm now using [ahk_explorer/ahk_explorer.ahk · FuPeiJiang/ahk_explorer](https://github.com/FuPeiJiang/ahk_explorer/blob/9a8b319769a693db25a069da7210b69cc328bd8f/ahk_explorer.ahk#L1) as a test in my AHK++ extension. I notice you have some code in a function immediately following a `return` statement, and I'm not sure why. AFAIK, that code will never be executed. But to be honest, I have written very little AHK code myself! Here is the snippet: [ahk_explorer/ahk_explorer.ahk:576-585](https://github.com/FuPeiJiang/ahk_explorer/blob/9a8b319769a693db25a069da7210b69cc328bd8f/ahk_explorer.ahk#L576-L585)

```ahk
return
sleep, 500

LV_GetText(OutputVar,A_EventInfo,2)
SplitPath, OutputVar, , , OutExtension, OutNameNoExt
if (OutNameNoExt) {
    Postmessage,0xB1, 0, % StrLen(OutNameNoExt), Edit2
} else {
    Postmessage,0xB1, 1, % StrLen(OutExtension)+1, Edit2
}
```

**Comment by FuPeiJiang** [comment, marked as answer]
1 of 3 ways to "comment" code
